config/promitor/resource-discovery/runtime.yaml

```diff
@@ -1,5 +1,7 @@
 server:
   httpPort: 88
+authentication:
+  mode: managedIdentity
```

Comment:
```suggestion
  mode: ManagedIdentity
```

Reply:
I used to use camel casing in Yaml configuration, but we can do it the way you want

Reply:
It's ok, as long as both are supported :D